megamek/src/megamek/client/ui/swing/util/FluffImageHelper.java

```diff
@@ -184,6 +184,9 @@ protected static Image loadFluffImage(final Entity unit) {
     private FluffImageHelper() { }
 
     private static String getImagePath(final ASCardDisplayable element) {
+        if ((element.getASUnitType() == ASUnitType.SV) && (element.hasMovementMode("a"))) {
+            return DIR_NAME_CONVFIGHTER;
+        }
```

Comment:
I'm really not a fan of this following the below. Can it instead be done as a check on case SV with a fall through to case CV?

Reply:
Yeah that works